From 7522545d973c8d33daac4e51fe6611276c702335 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Michael=20La=C3=9F?= <bevan@bi-co.net>
Date: Thu, 26 Dec 2024 14:32:16 +0100
Subject: [PATCH 3/3] hackrf-handler: fix wiring of biasT checkbox

47063973c2058eab663e4155193d6293e1ed38ba accidentally connected the
biasT checkbox to the RF amplifier.
---
 qt-devices/hackrf-handler/hackrf-handler.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/qt-devices/hackrf-handler/hackrf-handler.cpp b/qt-devices/hackrf-handler/hackrf-handler.cpp
index c30c9858..51d0704e 100755
--- a/qt-devices/hackrf-handler/hackrf-handler.cpp
+++ b/qt-devices/hackrf-handler/hackrf-handler.cpp
@@ -174,7 +174,7 @@
 	connect (this, &hackrfHandler::signal_antEnable,
 	         biasT_button, &QCheckBox::setChecked);
 	connect (this, &hackrfHandler::signal_ampEnable,
-	         biasT_button, &QCheckBox::setChecked);
+	         AmpEnableButton, &QCheckBox::setChecked);
 	connect (this, &hackrfHandler::signal_vgaValue,
 		 vgaGainSlider, &QSlider::setValue);
 	connect (this, &hackrfHandler::signal_vgaValue,
-- 
2.47.1

